packages/wallet-sdk/src/sign/SignRequestHandler.ts

```diff
@@ -105,6 +105,10 @@ export class SignRequestHandler implements RequestHandler {
       'wallet_addEthereumChain',
       'wallet_switchEthereumChain',
       'wallet_watchAsset',
+      'wallet_getCapabilities',
+      'wallet_sendTransaction',
+      'wallet_getTransactionStatus',
+      'wallet_showTransactionStatus',
```

Comment:
think we've landed on cutting wallet_showTransactionStatus. seems like it would be easy to add back if we need it but thinking we should remove here for now. lmk if im wrong though, if it would be too much of a headache to add back later then we can just leave it as is

Reply:
Sounds good, if we add it back dapps will have to update their code anyway to start using it. The only difference is just adding a version # bump to those codechanges

addressed in [35edf8f](https://github.com/coinbase/coinbase-wallet-sdk/pull/1138/commits/35edf8fa992029ab3feabc878b9891d7a7b0036f)